ftw/inflator/overrides.zcml

```diff
@@ -0,0 +1,26 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:browser="http://namespaces.zope.org/browser"
+    xmlns:i18n="http://namespaces.zope.org/i18n"
+    xmlns:zcml="http://namespaces.zope.org/zcml"
+    i18n_domain="ftw.inflator">
+
+    <browser:page
+        zcml:condition="not-have plone-5"
+        for="OFS.interfaces.IApplication"
+        name="plone-overview"
```

Comment:
Duplicate: https://github.com/4teamwork/ftw.inflator/pull/36/files#diff-5c3e52bda19d593850b74bcdb0ef46ceL25

I think this should be possible without an `overrides.zcml` at all. It did work on Plone 4; why shouldn't it work on Plone 5?

Reply:
Unfortunately browserlayers are not yet ready. 

The request only provides the following:
``` 
(Pdb) tuple(self.request.__provides__)
(<InterfaceClass zope.publisher.interfaces.browser.IDefaultBrowserLayer>, <InterfaceClass zope.publisher.interfaces.browser.IBrowserRequest>, <InterfaceClass zope.annotation.interfaces.IAttributeAnnotatable>)
```

and adding any sort of layer to this `browser:page` has no effect.

I tried `zope.publisher.interfaces.browser.IDefaultBrowserLayer` and a interface which inherits from `zope.publisher.interfaces.browser.IDefaultBrowserLayer`. Both had no effect.

Reply:
👍